inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_strided_slice_test.h

```diff
@@ -240,8 +240,16 @@ TEST_P(myriadLayersTestsStridedSlice_smoke, TestsStridedSlice) {
 
     // Load network.
     StatusCode st = GENERAL_ERROR;
+
+    std::map<std::string, std::string> config = {
+        { VPU_CONFIG_KEY(DETECT_NETWORK_BATCH), CONFIG_VALUE(NO) }
+    };
+    if (!CheckMyriadX()) {
+        config.insert({ VPU_CONFIG_KEY(DISABLE_REORDER), CONFIG_VALUE(YES) });
```

Comment:
Only style. I would prefer:
```
std::map<std::string, std::string> config;
config[VPU_CONFIG_KEY(DISABLE_REORDER)]= CheckMyriadX() ? CONFIG_VALUE(NO) : CONFIG_VALUE(YES);
```

But it looks like you want to emphasize the depreciation of M2, so this code looks ok to me

Reply:
But it's not equivalent. Actually for MyriadX we should also disable reordering but we don't need to set any option. It will be disabled by `hwOptimization` which `true` by default for MyriadX.

Reply:
Oh, I'm sorry)) It was too late)

Reply:
But actually it can be:  

```
std::map<std::string, std::string> config;
config[VPU_CONFIG_KEY(DETECT_NETWORK_BATCH)] = CONFIG_VALUE(NO);
config[VPU_CONFIG_KEY(DISABLE_REORDER)] = CheckMyriadX() ? CONFIG_VALUE(NO) : CONFIG_VALUE(YES);
```

Reply:
It doesn't make sense to set [`DISABLE_REORDER` , `NO`] for MyriadX because 1) We always want this option to be `YES`, as we don't need any reorder for MyriadX. 2) Whatever we set, reorder will be disabled for MyriadX anyway so it might confuse someone.